tools/TestGeneratedKeys.cpp

```diff
@@ -643,13 +643,13 @@ static bool testThresholdCryptosystem(
   // keys for this cryptosystem.
   for (uint16_t i = 0; i < numSigners; ++i) {
     IThresholdVerifier* verifier = verifiers[i];
-    if (verifier->getPublicKey().toString()
-          != referenceVerifier->getPublicKey().toString()) {
-      std::cout << "TestGeneratedKeys: FAILURE: Replica " << i << "'s key file"
-        " has the wrong public key for the " << name << " threshold"
-        " cryptosystem.\n";
-      return false;
-    }
+//    if (verifier->getPublicKey().toString()
```

Comment:
If we aren't using this code, let's delete it please

Reply:
I'll return the commented out code for this commit.
I have another test which I wrote which will enter in a subsequent commit